recipes/pyside/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "1.2.4" %}
+
+package:
+  name: pyside
+  version: {{ version }}
+
+source:
+  fn: PySide-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/P/PySide/PySide-{{ version }}.tar.gz
+  md5: 3cb7174c13bd45e3e8f77638926cb8c0
+  patches:
+    # eliminate spurious shiboken stderr output to avoid the 4MB log limit on Travis
+    - shiboken-osx-fix-stderr-output.patch
+    # https://github.com/sparsehash/sparsehash/commit/7b8afad35bd601ea780b0f72e215de96613d3c95
+    - fix-vc2013-hash_map.patch
+
+build:
+  number: 0
+  features:
+    - vc9               # [win and py27]
+    - vc10              # [win and py34]
+    - vc14              # [win and py35]
+  detect_binary_files_with_prefix: True
+
+  # https://bugreports.qt.io/browse/PYSIDE-321
+  # -> need to backport https://code.qt.io/cgit/pyside/pyside.git/commit/?id=23d77136da1f249f0a78c8c7e8124adc8c604230
+  skip: true  # [osx or (win and py35)]
+
+requirements:
+  build:
+    - toolchain
+    - python
+    - cmake
+    - qt 4.8*
+    - libxslt
+
+  run:
+    - python
+    - qt 4.8*
+    - libxslt
```

Comment:
[This]( http://upstream.rosalinux.ru/versions/libxslt.html ) may help. Also have asked ( https://github.com/lvc/upstream-tracker/issues/3 ) for it be added [here]( https://abi-laboratory.pro/tracker/ ).